src/test/test-strv.c

```diff
@@ -660,6 +660,46 @@ TEST(strv_extend_strv) {
         assert_se(strv_length(n) == 4);
 }
 
+TEST(strv_extend_strv_consume) {
```

Comment:
Please add a test case for `strv_extend_strv_consume(NULL, ...)`

Reply:
Hmm, that would literally be testing `assert()`? Do you mean to use `ASSERT_SIGNAL` really?